### DISCUSSION TITLE
Autocomplete Textarea Component

### DISCUSSION THREAD

**Discussion by louis030195** [assistant-ui/assistant-ui#198] [Ideas]
Great lib! I actually tweeted few weeks ago asking for components like this, perfect!

Would be cool to have drop-in autocomplete component like in Github Copilot / Gmail's / iPhone keyboard AI powered by LLM

https://github.com/vercel/ai/discussions/1965

**Comment by Yonom** [comment]
Thanks, this is a good suggestion!
